client/src/assets/styles/main.scss

```diff
@@ -82,6 +82,14 @@ a {
     background-color: $lightgrey;
   }
 
+  .theme--light.v-tabs-items {
+    background-color: $lightgrey;
+  }
+  
+  .theme--light.v-tabs > .v-tabs-bar {
+    background-color: $lightgrey;
+  }
```

Comment:
Is it necessary? I think it works with previous classes `.v-tabs-items` and `  .v-tabs > .v-tabs-bar` so I removed it